.tours/introducing-lwc-recipes.tour

```diff
@@ -0,0 +1,14 @@
+{
+  "title": "👋 Introducing LWC Recipes",
+  "steps": [
+    {
+      "directory": "src",
+      "description": "Recipes is a collection of easy-to-digest code examples for Lightning Web Components Open Source."
```

Comment:
"The recipes app" instead of just "Recipes" maybe?

Reply:
Done